microsoft-edge/webview2/howto/chromium-devtools-protocol.md

```diff
@@ -0,0 +1,194 @@
+1.  Type `Microsoft.Web.WebView2.DevToolsProtocolExtension` and choose **Microsoft.Web.WebView2.DevToolsProtocolExtension** > **Install**.   
+    
+:::image type="complex" source="./media/cdpnuget.png" alt-text="Ensure Microsoft.Web.WebView2.DevToolsProtocolExtension displays in the Visual Studio NuGet Package Manager" lightbox="./media/cdpnuget.png":::
+   Ensure **Microsoft.Web.WebView2.DevToolsProtocolExtension** displays in the Visual Studio NuGet Package Manager  
+:::image-end:::  
+
+## Step 4: Use DevTools Protocol Helper  
+
+1.  Add the `DevToolsProtocolExtension` namespace to your project.  
+    
+    ```csharp
+    using Microsoft.Web.WebView2.Core;
+    using Microsoft.Web.WebView2.Core.DevToolsProtocolExtension;
+    ```  
+    
+1.  Instantiate the `DevToolsProtocolHelper` object and navigate to `geolocation.html`.  
+    
+    ```csharp
+    async void InitializeAsync()
+    {
+        await webView.EnsureCoreWebView2Async(null);
+        DevToolsProtocolHelper helper = webView.CoreWebView2.GetDevToolsProtocolHelper(); 
+        
+        webView.CoreWebView2.Navigate(@"C:\{path\to\file}\geolocation.html");
+    }
+    ```  
+    
+1.  Run the [setGeoLocationOverrideAsync][GithubChromedevtoolsDevtoolsProtocolTotEmulationMethodSetgeolocationoverride] method.  For more information, navigate to [setGeolocationOverride][GithubChromedevtoolsDevtoolsProtocolTotEmulationMethodSetgeolocationoverride].  
+    
+    ```csharp
+    async void InitializeAsync()
+    {
+        await webView.EnsureCoreWebView2Async(null);
+        DevToolsProtocolHelper helper = webview.CoreWebView2.GetDevToolsProtocolHelper();
+        
+        webView.CoreWebView2.Navigate(@"C:\{path\to\file}\geolocation.html");
+        
+        // Latitude and longitude for Paris, France.
+        double latitude = 48.857024082572565;  
+        double longitude = 2.3161581601457386;  
+        double accuracy = 1;
+        await helper.Emulation.setGeolocationOverrideAsync(latitude, longitude, accuracy);
+        
+    }
+    ```  
+    
+1.  Run your app.  
+1.  To display the coordinates of Paris, France, choose the **Display Location** button.  
+    
+    :::image type="complex" source="./media/finallocation-cdp.png" alt-text="Display the .html file in a WebView2 control with the coordinates for Paris" lightbox="./media/finallocation-cdp.png":::
+       Display the `.html` file in a WebView2 control with the coordinates for Paris  
+    :::image-end:::  
+    
+## File a Chromium DevTools Protocol bug  
+
+The WebView2 team doesn't own the Chromium DevTools Protocol.  
+
+> [!IMPORTANT]
+> Direct feedback and bugs to the Chromium Issues repo.  
+
+To file a Chromium DevTools Protocol bug or issue, complete the following actions.  
+
+1.  File a [bug report][ChromiumBugsChromiumIssuesEntryComponentsPlatformDevtoolsPlatform].  
+1.  Navigate to [WebView Feedback][GithubMicrosoftedgeWebview2feedback] and open a new issue.  
```

Comment:
You have step one filing a chromium bug and step 2 filing a WebView2 bug? Is that intentional? Are you telling them to file a bug in both places? Or do you need to give instructions on when to file in one place or the other?

Reply:
My intention is for them to file in both places:
1) on the chromium bug - since that team owns the API
2) let us know in our feedback repo, so we can better understand how people are using the CDP package and what issues they are running into.